name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/TradeRepublicPDFExtractor.java

```diff
@@ -1437,7 +1437,7 @@ private void addAccountStatementTransaction_Format02()
         });
         this.addDocumentTyp(type);
 
-        Block depositRemovalBlock_Format01 = new Block("^[\\d]{2} [\\w]{3,4}([\\.]{1})?( [\\d]{4})? .berweisung .*$");
+        Block depositRemovalBlock_Format01 = new Block("^[\\d]{2} [\\wä]{3,4}([\\.]{1})?( [\\d]{4})? .berweisung .*$");
```

Comment:
In English, March then becomes March and equals 5 characters, so I would suggest that we change the date format for the months MMM or LLL as follows:

`[\\wä]{3,4}([\\.]{1})?`

to 

`(?:Jan(?:uar)?|Feb(?:ruar)?|M.r(?:z)?|Mar(?:ch)?|Apr(?:il)?|May|Mai|Jun(?:i|e)?|Jul(?:i|y)?|Aug(?:ust)?|Sep(?:tember)?|Oct(?:ober)?|Okt(?:ober)?|Nov(?:ember)?|Dez(?:ember)?|Dec(?:ember)?)\\.?)`

and the others too... then everything fits.
Great work!

Reply:
Thanks for feedback. I'm not sure if March will be shortened or not. I would wait for an English pdf debug to be able to cover it in test cases as well and keep your proposal in mind for that case but not change my pr for now.

BR Christian

Reply:
I think i change later in all pdf importer to this regulare expression if we needed